features/account/customer_account/viewing_orders_history/order_is_always_placed_in_a_base_currency_of_a_channel.feature

```diff
@@ -17,13 +17,13 @@ Feature: Order is always placed in a base currency of a channel
 
     @ui
     Scenario: Placing an order with other than base display currency
-        Given I changed my currency to "CAD"
+        Given I changed my currency to "GBP"
         And I had product "Angel T-Shirt" in the cart
         And I specified the shipping address as "Ankh Morpork", "Frost Alley", "90210", "United States" for "Jon Snow"
         And I proceed with "DHL" shipping method and "Cash on Delivery" payment
         When I confirm my order
         And I am viewing the summary of my last order
-        Then I should see "$40.00" as order's total
-        And I should see "$20.00" as order's subtotal
-        And I should see "$20.00" as item price
+        Then I should see "£40.00" as order's total
+        And I should see "£20.00" as order's subtotal
+        And I should see "£20.00" as item price
         And I should see "$40.00" as payment total
```

Comment:
Shouldn't it be also in `£`?

Reply:
no, because payments are processed in base currency